titus-server-gateway/src/main/java/com/netflix/titus/gateway/startup/TitusGateway.java

```diff
@@ -46,6 +50,14 @@ public static void main(String[] args) throws Exception {
             System.exit(1);
         }
 
+        try {
+            KubeApiConnector kubeApiConnector = new KubeApiConnector(Fabric8IOClients.createFabric8IOClient(), TitusRuntimes.internal());
```

Comment:
Do not use `TitusRuntimes.internal()`, but the `TitusRuntime` instance provided by Guice.